packages/common/src/HardhatConfig.js

```diff
@@ -42,7 +46,7 @@ function getHardhatConfig(configOverrides, workingDir = "./", includeTruffle = t
         "contracts/oracle/implementation/test/VotingTest.sol": LARGE_CONTRACT_COMPILER_SETTINGS,
       },
     },
-    ovm: { solcVersion: "0.8.4-broken_alpha" },
+    ovm: { solcVersion: "0.7.6" },
```

Comment:
I change this to let us use the OVM contracts, such as the imports like `import "@eth-optimism/...`